gitCoreJGit/src/main/java/com/virtuslab/gitcore/impl/jgit/GitCoreReflogEntry.java

```diff
@@ -0,0 +1,29 @@
+package com.virtuslab.gitcore.impl.jgit;
+
+import io.vavr.control.Option;
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+import org.eclipse.jgit.lib.ReflogEntry;
+
+import com.virtuslab.gitcore.api.IGitCoreCommitHash;
+import com.virtuslab.gitcore.api.IGitCoreReflogEntry;
+
+@EqualsAndHashCode
+@Getter
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+@ToString
```

Comment:
Can be simplified
```suggestion
@Data
@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
```
Lombok is smart enough to make ctor private anyway.

Reply:
The same probably in other places

Reply:
Done here and in `GitCorePersonIdentity`